### BIG differences, this is diff of wrong files.
### CLUS "Restart USE-VALUE"
### and CLHS topic selection http://clhs.lisp.se/Body/a_store_.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:restarts:use-value.txt	2017-04-27 21:59:59.000000000 +0100
+++ saved-pages-clus/cl:restarts:use-value.txt	2017-04-27 21:59:59.000000000 +0100
@@ -1,4 +1,25 @@
-USE-VALUE
-Please select which reference to USE-VALUE you intended
+Restart USE-VALUE

-FunctionRestart
\ No newline at end of file
+Data Arguments Required
+a value to use instead (once).
+Description
+the use-value restart is generally used by handlers trying to recover from errors of types such as cell-error, where the handler may wish to supply a replacement datum for one-time use.
+See Also
+{\secref\Restarts}, {\secref\InterfacesToRestarts}, Function INVOKE-RESTART, Function USE-VALUE, Function STORE-VALUE, Restart STORE-VALUE
+Notes
+No functions defined in this specification are required to provide a use-value restart.
\ No newline at end of file
